table.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet" href="table.css">
+    <title>HTML_table</title>
+  </head>
+
+  <body>
+    <table class="table">
+      <tr class="table__header-row">
+        <th class="table__header-row-body">Name</th>
+        <th class="table__header-row-body">Age</th>
+        <th class="table__header-row-body">Country</th>
+        <th class="table__header-row-body">FavouriteColor</th>
+      </tr>
+      <tr class="table__content-row">
+        <td class="table__content-row-body">Marc</td>
+        <td class="table__content-row-body">21</td>
+        <td class="table__content-row-body">Germany</td>
+        <td class="table__content-row-body">Blue</td>
+      </tr>
+      <tr class="table__content-row">
+        <td class="table__content-row-body">Bob</td>
```

Comment:
Sorry I should have seen that earlier:
```suggestion
        <td class="table__content-row-cell">Bob</td>
```

Reply:
This is a cell of the table not a body. You could argue that "table__cell" would suffice too.